frontend/locales/en/strings.json

```diff
@@ -1003,7 +999,7 @@
       "text": "Loading..."
     },
     "FileUploadButton": {
-      "label": "Chose file"
+      "label": "Choose file"
```

Comment:
Question: can't you delete this? I see it's only used by the feature this PR removes?
I may be wrong